source/dark-mode.txt

```diff
@@ -0,0 +1,32 @@
+.. _dark-mode:
+
+================
+Enable Dark Mode
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can configure |compass| to use a dark theme. To choose your theme,
```

Comment:
Hi @jeff-allen-mongo, this edit looks good. Wondering if we should add a "Starting in Compass 1.30.1" message to this page? I am running an older version of compass and don't have the theme tab.

Reply:
Sure. Since Compass provides auto-updates and visible prompts to the user to update, we've leaned away from including "Starting in Compass (x)" for features, but there's not really harm in including it.